src/main/java/spireMapOverhaul/zones/manasurge/events/ManaCycloneEvent.java

```diff
@@ -85,8 +84,7 @@ public ManaCycloneEvent() {
 
                     String chosenRelicId;
                     if (!availableRelics.isEmpty()) {
-                        int randomIndex = new Random().nextInt(availableRelics.size());
-                        chosenRelicId = availableRelics.get(randomIndex);
+                        chosenRelicId = availableRelics.get(AbstractDungeon.cardRng.random(availableRelics.size()));
```

Comment:
Events use miscRng for cards/relics/etc within them

Reply:
done